inst/tinytest/raw_data/Metamorpheus/ExperimentalDesign.tsv

```diff
@@ -0,0 +1,5 @@
+Run	Condition	BioReplicate	Fraction	Techrep
+NEG1-calib	neg	1	1	1
+NEG2-calib	neg	2	1	1

```

Comment:
This is something that Metamorpheus doesn't necessarily do correctly where they give different bioreplicates the same ID (1 & 2) - MSstats thinks of this as a paired design.  Change the ID.